packages/atlas/src/components/NftTileViewer/NftTileViewer.tsx

```diff
@@ -82,31 +64,29 @@ export const NftTileViewer: React.FC<NftTileViewerProps> = ({ nftId }) => {
       case 'TransactionalStatusIdle':
       case 'TransactionalStatusInitiatedOfferToMember':
         return {
-          ...nftCommponProps,
+          ...nftCommonProps,
           auction: 'none',
         }
       case 'TransactionalStatusBuyNow':
         return {
-          ...nftCommponProps,
+          ...nftCommonProps,
           auction: 'none',
           buyNowPrice: nft.transactionalStatus.price,
         }
       case 'TransactionalStatusAuction': {
-        const isEnded =
-          (nft.transactionalStatus.auction?.endedAtBlock || timeLeft <= 0) &&
-          nft.transactionalStatus.auction?.auctionType.__typename === 'AuctionTypeEnglish'
+        const isEnded = auctionPlannedEndBlock && getCurrentBlock() >= auctionPlannedEndBlock
```

Comment:
When an auction expires I still see `Less than the minute` slot and Im pretty sure that it worked before.